cpython/download_pypi_top.py

```diff
@@ -22,9 +22,17 @@
 import traceback
 import json
 import os
+import sys
 import time
 
-from termcolor import cprint
+
+try:
+    from termcolor import cprint
+except ImportError:
+    print("termcolor is missing, install it with: python -m pip install --user termcolor", file=sys.stderr)
```

Comment:
What do you think of adding one emtpy line for readability?

```suggestion
    print("Warning: termcolor is missing, install it with: python -m pip install --user termcolor", file=sys.stderr)
    print(file=sys.stderr)
```

Reply:
Looks good!